community/cypher/cypher-compiler-2.2/src/main/scala/org/neo4j/cypher/internal/compiler/v2_2/planner/QueryProjection.scala

```diff
@@ -148,11 +148,11 @@ final case class AggregatingQueryProjection(groupingKeys: Map[String, Expression
   def withShuffle(shuffle: QueryShuffle) =
     copy(shuffle = shuffle)
 
-  def exposedSymbols: Set[IdName] = (groupingKeys.keys ++  aggregationExpressions.keys).map(IdName.apply).toSet
+  override def exposedSymbols(qg: QueryGraph) = (groupingKeys.keys ++  aggregationExpressions.keys).map(IdName.apply).toSet
 }
 
 case class UnwindProjection(identifier: IdName, exp: Expression) extends QueryHorizon {
-  def exposedSymbols: Set[IdName] = Set(identifier)
+  override def exposedSymbols(qg: QueryGraph) = qg.allCoveredIds + identifier
```

Comment:
Why not compute the allCoveredIds when constructing the `UnwindProjection` in `UnwindConverter.addUnwindToLogicalPlanInput`?